x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -184,7 +194,7 @@ export const getDescriptionListContent = (
   const details = [
     {
       title: i18n.OPERATING_SYSTEM,
-      value: getOperatingSystems(exceptionItem._tags),
+      value: getOperatingSystems(exceptionItem._tags ?? []),
```

Comment:
In `kibana/x-pack/plugins/lists/common/schemas/request/create_exception_list_item_schema.ts` it looks like `_tags` gets defaulted to an empty array on creation, so we should be good to remove the `?? []`.